source/stylesheets/refills/_accordion-tabs.scss

```diff
@@ -55,21 +46,21 @@
   .tab-link {
     background-color: $tab-inactive-color;
     border-top: $tab-border;
-    color: $dark-gray;
+    color: $tab-link-color;
     display: block;
     font-weight: bold;
-    padding: $base-spacing/2 $gutter/2;
+    padding: ($tab-spacing/2) $gutter;
     text-decoration: none;
 
     @include media($tab-mode) {
       @include inline-block;
-      border-top-left-radius: $base-border-radius;
-      border-top-right-radius: $base-border-radius;
+      border-top-left-radius: $tab-border-radius;
```

Comment:
Properties should be ordered border-top, border-top-left-radius, border-top-right-radius